### DISCUSSION TITLE
Managing Helm Charts - Cannot find

### DISCUSSION THREAD

**Discussion by jackchuong** [goharbor/harbor#19758] [Q&A] [answered]
Hi all,
I'm Harbor newbie , I setup Harbor v2.8.3-19d69635 in my k8s internal cluster using Helm. It's working.
I read the docs here : https://goharbor.io/docs/1.10/working-with-projects/working-with-images/managing-helm-charts/
However I cannot find "Helm Charts" at the Harbor Interface
![Capture](https://github.com/goharbor/harbor/assets/86997250/84716161-8f29-4bf9-8ae5-cd670b9dcb27)
Please give me some advice, thank you very much.

**Comment by zyyw** [comment, marked as answer]
The doc you are referring to is `v1.10.x`, and the Harbor version you are using is v2.8.3. And chartmuseum has been removed from Harbor since v2.8.0:
* https://github.com/goharbor/harbor/issues/17958

**Reply by olivercp3** [reply]
In Harbor v2.8 above , so we only can push chart though `helm push`? I was wondering if there is another way to upload chart.

**Reply by zyyw** [reply]
Yes. In Harbor v2.8.0 and above, only oci helm chart is supported. How to push/pull oci helm chart:
* https://github.com/goharbor/website/pull/513

**Reply by olivercp3** [reply]
thank you, may I ask another question：
```shell
[root@vm]$  helm push cloudacademy-webapp-0.1.0.tgz oci://10.18.100.5:80/helm/
Error: failed to do request: Head "https://10.18.100.5:80/v2/helm/cloudacademy-webapp/blobs/sha256:ee8f1225c512e69a351a8a454dd714f6bcc3af83ef6c817d59b355457414a17f": http: server gave HTTP response to HTTPS client
```

my installation of Harbor uses HTTP rather than HTTPS, But seems like  `helm push` only support https?

**Reply by auguszero** [reply]
How do you solve it now？

**Comment by auguszero** [comment]
> thank you, may I ask another question：
> 
> ```shell
> [root@vm]$  helm push cloudacademy-webapp-0.1.0.tgz oci://10.18.100.5:80/helm/
> Error: failed to do request: Head "https://10.18.100.5:80/v2/helm/cloudacademy-webapp/blobs/sha256:ee8f1225c512e69a351a8a454dd714f6bcc3af83ef6c817d59b355457414a17f": http: server gave HTTP response to HTTPS client
> ```
> 
> my installation of Harbor uses HTTP rather than HTTPS, But seems like `helm push` only support https?

**Reply by olivercp3** [reply]
newest version `helm push` support HTTP by `--plain-http`
